exercises/pov/src/POV.hs

```diff
@@ -3,7 +3,7 @@ module POV (fromPOV, tracePathBetween) where
 import Data.Tree (Tree)
 
 fromPOV :: Eq a => a -> Tree a -> Maybe (Tree a)
-fromPOV = error "You need to implement this function."
+fromPOV node tree = error "You need to implement this function."
 
 tracePathBetween :: Eq a => a -> a -> Tree a -> Maybe [a]
-tracePathBetween = error "You need to implement this function."
+tracePathBetween nodeA nodeB tree = error "You need to implement this function."
```

Comment:
In `fromPOV` we only know that the element **may** be in the `Tree`, but `node` suggests two things:

1. It is part of a tree.
2. It is a node, not a leaf.

I would suggest simply `x`, because the meaning is already clear in the function's name.

In `tracePathBetween` we have a similar problem but, as @petertseng mentioned, one is supposed to be the starting element and the other the ending element, so a think a pair of related names would be great. Some suggestions:

- `start`, `end`
- `from`, `to`